scripts/Interface/ui/sans_isis/add_runs_page.py

```diff
@@ -6,14 +6,17 @@
 # SPDX - License - Identifier: GPL - 3.0 +
 from __future__ import absolute_import
 
-from PyQt4 import QtGui
-from PyQt4.QtCore import pyqtSignal
-from . import ui_add_runs_page
+from qtpy import QtWidgets
+from qtpy.QtCore import Signal
 
+from mantidqt.utils.qt import load_ui
 
-class AddRunsPage(QtGui.QWidget, ui_add_runs_page.Ui_AddRunsPage):
-    sum = pyqtSignal()
-    outFileChanged = pyqtSignal()
+Ui_AddRunsPage, _ = load_ui(__file__, "add_runs_page.ui")
```

Comment:
Why is this a global variable, not an instance variable?

Assigning it to self.ui = load_ui(__file__, "add_runs_page.ui") would make it a shorter call as well. Please a key difference is that it will only be loaded when needed and not when imported.